MeshLib/MeshEditing/ConvertToLinearMesh.cpp

```diff
@@ -58,6 +59,11 @@ std::unique_ptr<MeshLib::Mesh> convertToLinearMesh(MeshLib::Mesh const& org_mesh
             vec_new_eles.push_back(createLinearElement<MeshLib::Quad>(
                 e, vec_new_nodes));
         }
+        else if (e->getCellType() == MeshLib::CellType::HEX20)
+        {
+            vec_new_eles.push_back(createLinearElement<MeshLib::Hex>(
+                e, vec_new_nodes));
+        }
```

Comment:
Maybe the code could be more readable if a `switch case` statement would be used here?! If you do not agree let the code as it is.